packages/UI/src/components/Dialog/Dialog.stories.tsx

```diff
@@ -78,11 +81,14 @@ export const DefaultWithNoClose: Story = {
           <DialogContent hasCloseBtn={false}>
             <DialogHeader>
               <DialogTitle>
-                <div className="h-8 w-96 bg-white uppercase" />
+                {/* eslint-disable-next-line tailwindcss/no-arbitrary-value */}
+                <div className="h-8 w-[448px] bg-white uppercase" />
               </DialogTitle>
             </DialogHeader>
-            <div className="h-36 w-96 bg-grayscale-100"></div>
-            <div className="h-8 w-96 bg-white"></div>
+            {/* eslint-disable-next-line tailwindcss/no-arbitrary-value */}
+            <div className="h-36 w-[448px] bg-grayscale-100"></div>
+            {/* eslint-disable-next-line tailwindcss/no-arbitrary-value */}
```

Comment:
eslint-disable-next-line tailwindcss/no-arbitrary-value should be avoided as much as possible, just remove it.

eslint-disable-next-line tailwindcss/no-arbitrary-value means I can't notice this viloation when I run `yarn workspace @devlaunchers/components lint` which ultimately means the design tokens won't get updated and other devs will keep using arbitrary-value in their apps.

The only way I know developers are struggling with finding a token is through tailwindcss/no-arbitrary-value so disabling it makes me blind to what's missing and subsequently makes Anouar and Design Team blind about this missing token!